### ISSUE TITLE
I use conditional GAN to train my model ,but i don not know how to generate 3Dimage from trained model

### ISSUE THREAD

**Issue by zzx721838322** [batmanlab/HA-GAN#11]
when i use visualization.ipynb, it occored the problem :
Model_HA_GAN_256.py  -> class Generator            
    def forward(self, h, crop_idx=None, class_label=None):
        if crop_idx != None or self.mode=='eval':
            if self.num_class > 0:
                h = torch.cat((h, class_label), dim=1)
class_label should not be None.
but i set class_label = 0 or tensor([0]）,it still unable to work
please tell me what should i do

**Comment by zzx721838322** [comment]
i found how to solve this problem... thank you
